src/scenes/App/index.tsx

```diff
@@ -58,7 +58,23 @@ const App = () => {
 
   return (
     <div className="app">
-      <Helmet defaultTitle={formatMessage({ id: 'appTitle' })} />
+      <Helmet
+        defaultTitle={formatMessage({ id: 'appTitle' })}
+        link={[
+          {
+            rel: 'shortcut icon',
+            href: partnerCompany
+              ? `src/ext/${partnerCompany}/assets/favicon.ico`
+              : '/favicon.ico',
+          },
+          {
+            rel: 'manifest',
+            href: partnerCompany
+              ? `src/ext/${partnerCompany}/assets/manifest.json`
+              : '/manifest.json',
+          },
+        ]}
+      />
```

Comment:
This could also be instead handled at a build time, but felt easier to add this kind of thing